app/models/webhooks/slack/synapse_removed_from_map.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class Webhooks::Slack::SynapseRemovedFromMap < Webhooks::Slack::Base
+  def text
+    connector = eventable.desc.empty? ? '->' : eventable.desc
+    # todo express correct directionality of arrows when desc is empty
+    "\"*#{eventable.topic1.name}* #{connector} *#{eventable.topic2.name}*\" was removed by *#{event.user.name}* as a connection from the map *#{view_map_on_metamaps}*"
+  end
+end
```

Comment:
Every webhook we set up in the db subscribes to whichever events out of all of these that it wants. So it's configurable. You could listen for all events, or just one